components/inputs/input-number.js

```diff
@@ -0,0 +1,83 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { getUniqueId } from '../../helpers/uniqueId.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { inputLabelStyles } from './input-label-styles.js';
+import { inputStyles } from './input-styles.js';
+import { offscreenStyles } from '../offscreen/offscreen.js';
+
+class InputNumber extends LitElement {
+
+	static get properties() {
+		return {
+			autocomplete: { type: String },
+			autofocus: { type: Boolean },
+			disabled: { type: Boolean },
+			label: { type: String },
+			labelHidden: { type: Boolean, attribute: 'label-hidden' },
+			max: { type: Number },
+			min: { type: Number },
```

Comment:
I think we also want to make these `String`s since `min` and `max` are string properties on `input`

Reply:
Hmm. `max` and `min` on the generic input are weird though because they need to handle dates and times. In our case we know this is a number, so couldn't we make it a number and then convert it to a string when we set it on the input?
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/max

Reply:
I couldn't conclusively find if there were any special values allowed for `type="number"` that couldn't be represented with `Number` so figured it'd be safer than later finding out there is one that someone needs and then having to change it from `Number` to `String` once out in the wild.

But if we're not worried about that I'm fine with keeping it as a `Number`.

Reply:
I'm not worried, especially since the table in the MDN docs lists `<Number>` as the syntax for numeric inputs. The HTML spec basically says that if the string parses to a number, then that's the minimum/maximum.

Reply:
Scientific notation may be worth checking, both for the attributes, and also for the user specified value.